README.md

```diff
@@ -8,6 +8,20 @@ infrastructure.
 Setup
 -----
 
+### Setting up a GitHub PAT
+
+As a prerequisite for running all scripts in this repo, you will need a GitHub Personal Access
+Token with full access to your account. You can generate one
+[here](https://github.com/settings/tokens).
+
+While creating the token, make sure at the least the scopes `repo`, and `workflow`.
+If you have the rights to manage Biomage repositories, the `admin:org`, `admin:public_key`,
+`admin:repo_hook` and `admin:org_hook` scopes must also be checked for those commands to work. 
```

Comment:
can you clarify on how does one know if they have the rights to manage Biomage repositories? I imagine that a lot of people will be confused at that point, wondering whether they should add these scopes or not.